core/src/test/java/io/questdb/test/griffin/engine/functions/regex/ILikeFunctionFactoryTest.java

```diff
@@ -26,15 +26,31 @@
 
 import io.questdb.cairo.sql.RecordCursor;
 import io.questdb.cairo.sql.RecordCursorFactory;
-import io.questdb.test.AbstractGriffinTest;
 import io.questdb.griffin.SqlException;
 import io.questdb.std.Chars;
+import io.questdb.test.AbstractGriffinTest;
 import io.questdb.test.tools.TestUtils;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class ILikeFunctionFactoryTest extends AbstractGriffinTest {
 
```

Comment:
If test is meant to mimic pg-wire behavior with bind vars then it'd be a bit better to use indexed bind variable .

Reply:
The goal is to cover behavior with bind variable expression for all possible cases. Added missing tests for indexed vars in bd5e0adc1191def83438f8452463f0e139aa1745